README.md

```diff
@@ -467,10 +477,10 @@ Script
 ---
 
 The script unit is composed of 3 units:
-* `script.c` - integration of scripts with Redis (commands execution, set replication/resp, ...)
-* `script_lua.c` - responsible to execute Lua code, uses `script.c` to interact with Redis from within the Lua code.
+* `script.c` - integration of scripts with Redis Community Edition (commands execution, set replication/resp, ...)
```

Comment:
i think many of these are excessive (this one being a good example).
i.e. if you run a blind search and replace for the second time, you'll get `Redis Community Edition Community Edition`.

i think it's sufficient to replace the ones at the top that describe what this Repo is, but there's no need to replace each and every mention of redis, specifically in the technical parts (the part of this file that describes the codebase)

Reply:
Everything we write here and assert here applies only to Redis Community Edition. "Redis" is our commercial offer, so we shouldn't mention it here.

Reply:
i understand that now the "Redis" is the commercial one. but you can't hope to remove every mention of the string "redis" from this project, besides in the context of this file "Redis" can be just short for "Redis Community Edition".

the parts that describe the commands, the protocol obviously apply to the commercial one too, and the parts that describe the source code directory structure are silly to replace IMHO.
if you do that, then why not replace all the other references in the code.
e.g. test_helper.tcl starts with `# Redis test suite.`.

if you're concerned that you just don't want this in the main md file of the project, then maybe we should move the part that discusses the source code to another file.. but then i wonder why you changed TLS.md and not tests/README.md?

Reply:
There are two paths we can take:

- Everyone looking at this code on GitHub should only care about Redis Community Edition. They don't and shouldn't care if it applies to our commercial offers or not. Hence, everything should be "Redis Community Edition".
- We can leave "Redis" when it applies to both our community and commercial offers, but that means we should analyze each mention.

I think we should take the first path.

Regarding "Redis" as a short for "Redis Community Edition" - I agree, but I think we should do that only in the code, but not in the textual files.

Reply:
is `tests/README.md` code or textual file? why is it different than `TLS.md`? or the bottom half of `README.md` (which clearly is about code)?

Reply:
Updated `tests/README.md` as well.
Made some additional updates to `README.md`, but I left 'Redis commands', 'Redis protocol', etc.

Reply:
i'm arguing that this mindless search and replace is wrong (not that you should do more of it). i brought up `tests/REDME.md` as an argument for you to revert the changes to `TLS.md` and much of the changes in the root `REAME.md`
@yossigo please comment.

Reply:
Definitely not mindless. If there are more important files - let me know.